### PR TITLE
Release note iOS 2023.12

### PR DESCRIPTION
**Pull request by bgoncal**
## Proposed change

Release note iOS 2023.12

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+comments: true
+categories: iOS, Release-Notes
+og_image: /images/blog/2023-12-ios-lets-go/ios-og.png
+---
+
+We have good news for those hoping for more features and faster development of the iOS Companion App! I, Bruno Pantaleão, have joined Nabu Casa as an iOS engineer (of course, also working on watchOS, iPadOS, and macOS apps), and am planning to continue the great work done by Robbie Trencheny, Zac West, and the community, giving the iOS app the attention that it deserves. So let’s talk about this month’s 2023.12 release of the app!
+
+![Companion App for iOS 2023.12](/images/blog/2023-12-ios-lets-go/ios-og.png)
+
+## Apple Thread network credentials sharing
```
Comment (balloob) [inline source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown:19]:
```suggestion
<!--more-->

## Apple Thread network credentials sharing
```

File: source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Companion app for iOS 2023.12: Let’s go!"
+description: "Bruno Pantaleão has joined Nabu Casa as an iOS engineer and will work on improving the experience for Companion App for iOS users. 2023.12 brings us Apple Thread credentials sharing, a new WatchOS icon, a Select All scene toggle, and a What’s New link."
```
Comment (c0ffeeca7) [inline source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown]:
```suggestion
description: "Bruno Pantaleão has joined Nabu Casa as an iOS engineer and will work on improving the experience of the Companion App for iOS users. 2023.12 brings us Apple Thread credentials sharing, a new WatchOS icon, a **Select All** scene toggle, and a **What’s New** link."
```

File: source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Toggle all scenes for watchOS
+
+Previously, you would have to disable scenes one by one to hide them on your Apple Watch. If you, like me, have a Philips Hue bridge, you would probably see several auto-generated scenes (like ‘Bathroom concentrate’ and ‘Bathroom Arctic aurora’ below :D ) that are not always relevant to see on your watch. We added a button to quickly toggle between all scenes now in the iOS companion app.
+
+![Toggle all scenes for watchOS](/images/blog/2023-12-ios-lets-go/3.png)
+
+## Keep up with “What’s new” in the App
+
+We have added a “What’s new?” link in companion App settings so you can quickly access the latest App release notes.
```
Comment (c0ffeeca7) [inline source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown]:
```suggestion
We have added a **What’s new?** link in companion App settings so you can quickly access the latest App release notes.
```

File: source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+![Keep up with Whats new in the App](/images/blog/2023-12-ios-lets-go/4.jpeg)
+
+## Support for iOS 12, 13 and 14 in 2024.01
```
Comment (c0ffeeca7) [inline source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown]:
```suggestion
## Support for iOS 12, 13, and 14 in 2024.01
```

File: source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+In the new year, we will do some housecleaning to make sure that the iOS Companion App is prepared for the future. To do so, we will stop supporting iOS 12, 13, and 14 in release 2024.01 of the Companion App. We know our users are repurposing older phones and tablets into dashboards and controls for their homes, which aligns with our focus on sustainability. It's why we try to keep our apps running on older devices for as long as we can.
+
+Currently, less than 1% of our users (According to the App Store analytics provided by users who gave permission to share their data with Apple) are still on these three iOS versions, while supporting them is making the codebase hard to maintain and blocks us from using newer iOS features. This change will make it easier for new contributors to feel comfortable contributing to the iOS codebase, which is also one of my goals. With a more modern codebase, we can give more attention to PRs and help other contributors have everything they need to feel comfortable submitting PRs.
```
Comment (c0ffeeca7) [inline source/_posts/2023-12-27-companion-app-for-ios-202312-lets-go.markdown]:
```suggestion
Currently, less than 1% of our users (according to the App Store analytics provided by users who gave permission to share their data with Apple) are still on these three iOS versions. Supporting them makes the codebase hard to maintain and blocks us from using newer iOS features. This change will make it easier for new contributors to feel comfortable contributing to the iOS codebase, which is also one of my goals. With a more modern codebase, we can give more attention to PRs and help other contributors have everything they need to feel comfortable submitting PRs.
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @bgoncal :+1:

**Comment by frenck** [comment]
Drafting this PR, as it is aimed for the 27th (to prevent accidental merge).

**Pull request merged**